sdk/errors/errors.go

```diff
@@ -80,3 +82,17 @@ func IsErrorConnectionReset(err error) bool {
 	}
 	return errors.Is(err, syscall.ECONNRESET)
 }
+
+// IsErrorTimedOut verifies error
+func IsErrorTimedOut(err error) bool {
+	if runtime.GOOS == "windows" {
```

Comment:
Better to move "windows" to const

Reply:
actually, never seen such examples on github)